### PR TITLE
{chem}[intel/2017b] Amber v16 with AmberTools 17

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
New Amber v16 with AmberTools 17. Needs ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1305~~

### CODE REVIEW

File: easybuild/easyconfigs/a/Amber/Amber-16-intel-2017b-AmberTools-17-patchlevel-0-7.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+dependencies = [
+    ('netCDF', '4.4.1.1'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('Python', '2.7.14'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+]
+
+runtest = False
```
Comment (boegel) [inline easybuild/easyconfigs/a/Amber/Amber-16-intel-2017b-AmberTools-17-patchlevel-0-7.eb]: @akesandgren The existing `Amber` easyconfigs don't have this, why is this needed?

Comment (akesandgren) [reply easybuild/easyconfigs/a/Amber/Amber-16-intel-2017b-AmberTools-17-patchlevel-0-7.eb]: Needed and needed.
Th extra dependencies are there because Amber searches for them and can use them.

The patches I haven't had time to push upstream yet but are important for a correct build.

Comment (boegel) [reply easybuild/easyconfigs/a/Amber/Amber-16-intel-2017b-AmberTools-17-patchlevel-0-7.eb]: @akesandgren I was referring to `runtest = False` :)

Comment (akesandgren) [reply easybuild/easyconfigs/a/Amber/Amber-16-intel-2017b-AmberTools-17-patchlevel-0-7.eb]: Ahhh, can't remember at the moment, will have to check...

Comment (akesandgren) [reply easybuild/easyconfigs/a/Amber/Amber-16-intel-2017b-AmberTools-17-patchlevel-0-7.eb]: Found some bugs still remains to be solved. This easyconfig is not complete, there is at least one patch file missing.
Plus some failures during tests.

Will try to figure out the test failures.

**Comment by migueldiascosta** [comment]
@akesandgren can this be closed in favour of #6618 ?

**Comment by akesandgren** [comment]
Yeah, I'll close it.
